enterprise/releases/2023-07-0.mdx

```diff
@@ -0,0 +1,80 @@
+    - Note that you may see small differences in top-level percentile numbers going forward compared to historic calculations
+
+
+## Bugfixes
+
+* Config Settings
+  - Move autovacuum_vacuum_insert_threshold and autovacuum_vacuum_insert_scale_factor from Other Settings config group to Autovacuum config group <!-- PR 2905 -->
+  - Update and sync Postgres setting config group with Postgres 15 <!-- PR 2917 -->
+* Query and table overview: Fix edge case that accidentally counted the same timeseries data point twice <!-- PR 2962 -->
+  - This edge case only occurred when viewing a very short time frame (2 hours or less), within certain conditions. If the error occurred, you might have seen the "% of Total Runtime" sum up to more than 100%, or other summary statistics to be higher than what you would get when viewing the detail statistics individually.
+* Index Advisor
+  - Fix incorrect method/costs shown for scans on query details <!-- PR 2959 -->
+    - This bug caused the scan list to pull in costs and methods from other queries in the same database on the same table, that had an identical scan (i.e. same set of WHERE/JOIN conditions)
+    - Whilst this information was not utilized for index recommendations, it may have been misleading when reading the scan information on the query details page.
+* Alerts & Check-Up
+  * Run each daily check in a separate background jobs for each server <!-- PR 2907 -->
+    - This improves resiliency against crashes in individual checks
+* Object metadata retention
+  * Fix accidental early deletion of invalidated objects <!-- PR 2970 -->
+    - This fixes a bug that caused invalidated objects to be deleted ahead of retention time, potentially causing an error when visiting a page for an invalidated object. Invalidated objects mean that they are no longer present on the database server, such as deleted tables. This affected only the metadata of these objects, not the statistics.
+  * Mark databases that are no longer visible in a snapshot as "invalidated" <!-- PR 2895 -->
+    - Previously such databases would never be deleted in pganalyze (despite having been removed on the actual database)
+    - Deleted databases will now disappear from lists immediately (e.g. the "All databases" list, and the database dropdown), but direct links will continue to work for the remainder of the retention time
+* Snapshot processing
+    * Require collector API keys for snapshot API, do not accept read or read/write keys <!-- PR 2847 -->
+    * Periodically call malloc_trim to reduce peak memory consumption <!-- PR 2939 -->
+    * Remove automatic restart of snapshot workers every 6 hours <!-- PR 2954 -->
+      - In the past this was necessary due to instability of the workers running over a long time - in our testing these problems are resolved, and turning off the restarts will avoid edge cases where the old behaviour caused snapshots to be partially processed
```

Comment:
I think we tend to use American English, so maybe we want to use "behavior" here instead